npm-audit/package.json

```diff
@@ -82,8 +82,8 @@
     "eslint-plugin-import": "^1.16.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.3.0",
-    "jest": "20.0.3",
-    "jest-cli": "20.0.3",
+    "jest": "^23.0.0",
+    "jest-cli": "^23.0.0",
```

Comment:
is the `^` intended ?

Reply:
Yes I trust the semver on jest.

Reply:
Yeah I think we can trust facebook to follow semver